python/sglang/srt/models/llava.py

```diff
@@ -27,7 +30,7 @@
 from sglang.srt.models.llama2 import LlamaForCausalLM
 from sglang.srt.models.mistral import MistralForCausalLM
 from sglang.srt.models.qwen2 import Qwen2ForCausalLM
-
+import math
```

Comment:
fix imports and lint errors https://github.com/sgl-project/sglang/blob/main/docs/en/contributor_guide.md#format-your-code